.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,40 @@
+name: continuous-integration
+on: [pull_request]
+jobs:
+  test-runner:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '12'
+        check-latest: true
+    - name: Install prerequisites
+      run: | 
+       sudo apt-get install -y build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libglu1-mesa-dev freeglut3-dev mesa-common-dev libcairo2-dev xserver-xorg-dev libxext-dev g++-4.8 xvfb
```

```suggestion
       sudo apt-get install -y build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libglu1-mesa-dev freeglut3-dev mesa-common-dev libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev  xserver-xorg-dev libxext-dev g++-4.8 xvfb
```

https://github.com/Automattic/node-canvas/wiki/Installation:-Ubuntu-and-other-Debian-based-systems